src/main/java/com/alipay/oceanbase/rpc/protocol/packet/ObRpcPacketHeader.java

```diff
@@ -121,7 +122,7 @@ public ObRpcPacketHeader() {
         #define OB_LOG_LEVEL_TRACE 4
         #define OB_LOG_LEVEL_DEBUG 5
          */
-        flag = 0x7; // let ObServer determine the ob log level.
+        flag = 0x7 | IS_KV_REQUEST_FALG; // let ObServer determine the ob log level.
```

Comment:
Make it clear, 0x7 -> ObServer determine the ob log level

Reply:
done